src/control/lib/control/pool.go

```diff
@@ -610,6 +611,24 @@ func PoolQuery(ctx context.Context, rpcClient UnaryInvoker, req *PoolQueryReq) (
 	return pqr, convertMSResponse(ur, pqr)
 }
 
+func UpdatePoolQueryState(pqr *PoolQueryResp) {
+	// Pool state is unknown as default, if TotalTargets is 0.
+	if pqr.TotalTargets == 0 {
+		pqr.State = "Unknown"
```

Comment:
also you could probably return here

Reply:
This shouldn't be exposed as a new API function. Instead, it could be a method on the `PoolQueryResp` struct, e.g. `.UpdateState()`. You could then call the method in `mgmtSvc.PoolQuery()` and also in `daos.poolQueryCmd.Execute()` to avoid duplicated logic.